lib/machinery_helper.rb

```diff
@@ -34,17 +31,25 @@ def initialize(s)
     @system = s
     @arch = @system.arch
 
-    @local_helpers_path = "/usr/share/machinery/helpers"
+    @local_helpers_path = File.join(Machinery::ROOT, "machinery-helper")
+    @remote_helpers_path = File.join(Machinery::REMOTE_HELPERS_PATH, "machinery-helper")
   end
 
   def local_helper_path
-    File.join(@local_helpers_path, @arch, "machinery-helper")
```

Comment:
This would make the logic more complicated without much gain for now and we decided to not share helpers between different architecture so there would be no use for it.